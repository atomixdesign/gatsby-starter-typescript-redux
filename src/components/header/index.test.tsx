import React from 'react'
import { mount, render } from 'enzyme'
import Header from './index'

it('renders the header', () => {
  const wrapper = render(<Header siteTitle="Welcome to Gatsby" />)

  expect(wrapper).toMatchSnapshot()
})

it('renders the header with title', () => {
  const wrapper = mount(<Header siteTitle="Welcome to Gatsby" />)

  expect(wrapper.find('h1').text()).toBe('Welcome to Gatsby')
})
