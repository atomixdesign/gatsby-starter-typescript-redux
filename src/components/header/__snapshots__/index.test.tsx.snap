// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the header 1`] = `
<header
  style="background:rebeccapurple;margin-bottom:1.45rem"
>
  <div
    style="margin:0 auto;max-width:960px;padding:1.45rem 1.0875rem"
  >
    <h1
      style="margin:0"
    >
      <a
        href="/"
        style="color:white;text-decoration:none"
      >
        Welcome to Gatsby
      </a>
    </h1>
  </div>
</header>
`;
